demo/apiExamples.md

```diff
@@ -33,15 +33,13 @@ The auro-pane component displays shoulder date information.
 
 ### Basic
 
-<div class="twoColDemoRow">
-  <div>
-    <div class="exampleWrapper">
-      <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basicSingle.html) -->
-      <!-- The below content is automatically added from ./../../apiExamples/basicSingle.html -->
-      <auro-pane date="2020-09-20"></auro-pane>
-      <!-- AURO-GENERATED-CONTENT:END -->
-    </div>
-<auro-accordion lowProfile justifyRight>
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basicSingle.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/basicSingle.html -->
+  <auro-pane date="2020-09-20"></auro-pane>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion alignRight>
```

Comment:
None of the links in the API examples link to content in the page.

Reply:
We need to get the SSR work done and this isn't an issue related to that. I asked Jordan to make sure lines 36 and 37 were removed on the repos we are touching because they are part of why the accordions are broken. Otherwise, I think documentation changes should be addressed in new tickets so we can get this SSR work done this week.